.github/workflows/build-and-test.yml

```diff
@@ -21,10 +21,29 @@ jobs:
 
     - name: Check if the page builds
       run: npm run-script build
+      
+    - name: Zip build
+      run: zip -r public.zip public -x "public/assets/screenshots/*" "public/assets/img/blog/*" "public/assets/img/science/*"
+      
+    - name: Upload artifact
+      uses: actions/upload-artifact@v3
+      with:
+        name: public
+        retention-days: 1
+        path: public.zip
 
   test:
     needs: build
     runs-on: ubuntu-latest
+    env:
+      NO_COLOR: true
+      TERM: xterm
```

Comment:
Since these environment variables are only used by Cypress, it would be more readable to add them to the step which calls Cypress at the bottom of the workflow where `ELECTRON_EXTRA_LAUNCH_ARGS` is defined.

It's an optional change. The workflow works fine the way it is.

Reply:
I moved them down to the Cypress step in https://github.com/corona-warn-app/cwa-website/pull/3389/commits/9bf3a0258a53e938eb11b94c0a3cdf574a285e51, thanks for the suggestion